faithcomesbyhearing/dbp
# FCBH-1384 Improve v2 syncing efficiency

# Description
- Updated kernel functions in order to improve the  v2 syncing efficiency
   - Users 
   - Profiles
   - Bookmarks 
   - Notes
   - Highlights

## Issue Link
[FCBH-1384](https://fullstacklabs.atlassian.net/browse/FCBH-1384) 

## How Do I QA This
- If you want can add `V2_V4_SYNC_CHUNK_SIZE` to the `.env` on your local environment to change the default sync chunk size of `5000`

- Run the commands
    1. `php artisan syncV2:users`
    2. `php artisan syncV2:profiles`
    3. `php artisan syncV2:bookmarks`
    4. `php artisan syncV2:highlights`
    5. `php artisan syncV2:notes`
- Verify that now the v2 data synchronization is working faster

## Checklist
